### DISCUSSION TITLE
Adicionar forma de pagamento

### DISCUSSION THREAD

**Discussion by aqalmeida** [RamonSilva20/mapos#1817] [Q&A] [answered]
Bom dia.
Como faço para adicionar forma de pagamento?
Tenho parceiros e alguns clientes não preciso gerar boleto, apenas uma data para pagemento e eles fazem pix ou transferência.
Queria ter um controle.
Ex. Pagamento para daqui 10 dias.
Aparece na área de cobrança.
Verifico se pagou e confirmo o pagamento.
Agradeço desde já pela atenção.

**Comment by SuperPorte** [comment, marked as answer]
Já tem isso em lançamentos. Quando você fatura uma OS, pode deixar ela com a caixa de []Pago desmarcada.
Legal mesmo seria aceitar mais de 1 pagamento. Tipo: OS = a R$500
PIX = 250
credito = 200
dinheiro = 50

**Reply by aqalmeida** [reply]
Sim mas só por filtro em lançamentos para pegar o pagamento pendente e tem que escolher data inicial e final.
Não aparece em cobrança.

**Reply by SuperPorte** [reply]
Não entendi sua necessidade então....

![image](https://user-images.githubusercontent.com/68247011/168678186-f64c3970-8352-47bf-afb8-f07b0563e2bf.png)

Não é isso?
![image](https://user-images.githubusercontent.com/68247011/168678288-8889fce6-468b-4cbe-86ce-3c00e4f29f68.png)

**Reply by aqalmeida** [reply]
![image](https://user-images.githubusercontent.com/105503624/168678500-32750d98-ea4a-4ba7-99ff-29065e8843fe.png)
Não aparece financeiro aqui

**Reply by SuperPorte** [reply]
Mas não é financeiro, é lançamentos

**Reply by aqalmeida** [reply]
![image](https://user-images.githubusercontent.com/105503624/168678864-4fbffd9d-3720-4d0a-b2dd-7015182cbbfd.png)
quando marco na OS não recebido deveria cair  aqui.

**Reply by SuperPorte** [reply]
Aí é para "ferramenta" de cobranças. Tipo MercadoPago, Asaas, etc.
Todo movimento de CAIXA (pagamentos de contas, fornecedores, recebimentos, ...) fica em lançamentos.

**Reply by aqalmeida** [reply]
entendi.
obrigado pelo esclarecimento.

**Reply by SuperPorte** [reply]
:D As ordens hehe
Entra no nosso grupo do Telegram =D

**Reply by itiossi** [reply]
@bernarcom Você poderia por gentileza nos enviar o link do grupo do telegram?
